telegrambot/migrations/0010_userprivilege_can_superban_members.py

```diff
@@ -0,0 +1,19 @@
+# Generated by Django 3.2.6 on 2021-08-14 20:07
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('telegrambot', '0009_groupmembership_status'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='userprivilege',
+            name='can_superban_members',
+            field=models.BooleanField(default=False, help_text='True, if the administrator can ban a member from all groups of the network. This should be setonly for the members of the C.A.N.'),
```

Comment:
`This should be setonly for the members of the C.A.N.`
Add a space between `set` and `only`